pool/bit_math.gno

```diff
@@ -4,103 +4,104 @@ import (
 	"gno.land/p/demo/ufmt"
 
 	"gno.land/r/demo/consts"
+
+	"gno.land/p/demo/u256"
 )
 
-func bitMathMostSignificantBit(x bigint) uint8 {
-	require(x > 0, ufmt.Sprintf("[POOL] bit_math.gno__bitMathMostSignificantBit() || x(%s) > 0", x))
+func bitMathMostSignificantBit(x *u256.Uint) uint8 {
+	require(x.Gt(u256.NewUint(0)), ufmt.Sprintf("[POOL] bit_math.gno__bitMathMostSignificantBit() || x(%s) > 0", x))
```

Comment:
```
const ZERO = u256.newUint(0)
x.Gt(u256.ZERO)
```

@notJoon which one do you prefer ??

Reply:
I prefer the x.Gt version.

Reply:
then perhaps consts should include `const ZERO` too just like `ZERO_ADDRESS`

Reply:
Currently `u256.Uint` is always used in pointer form, so exposing the ZERO as a const would cause a security issue. We have talked about using Uint as struct, not pointers, and in that case this would not be a problem. Do you guys think we should switch over to non-pointer convention?

Reply:
nah I think its fine